public/js/angular/controllers.js

```diff
@@ -1,6 +1,6 @@
 angular
-  .module('webmakerApp')
-  .controller('navigationController', ['$scope', '$location', '$routeParams', '$rootScope', 'weblit', 'wmNav', 'CONFIG',
+  .module("webmakerApp")
```

Comment:
Then we should change the jscsrc file in mofostyle to have:

```
"validateQuoteMarks": "'"
```

I can make that locally and update the quotes here, but to enforce it we need to update the jscsrc file.